server/middleware/getFormData.js

```diff
@@ -1,9 +1,10 @@
 module.exports = formService => async (req, res, next) => {
   try {
-    const formData = await formService.getFormResponse(req.user.username)
+    const { bookingId } = req.params
```

Comment:
as we need to access request params, moving this from being router level to route level middleware